appveyor.yml

```diff
@@ -39,14 +37,14 @@ test_script:
   - yarn test
 #  - yarn test-e2e
   - yarn package-dev
-  - 7z a development.zip release
+#  - 7z a development.zip release
```

Comment:
This is actually a call to the CI box's installed 7zip. The package `electron-devtools-installer` depends on the `7zip` package, so we can't get rid of it.